Resources/Prototypes/Tiles/floors.yml

```diff
@@ -190,6 +190,42 @@
   itemDrop: FloorTileItemSteel
   heatCapacity: 10000
 
+- type: tile
+  id: FloorBrassFilled
+  name: tiles-brass-floor-filled
+  sprite: /Textures/Tiles/Misc/clockwork/clockwork_floor_filled.png
+  baseTurf: PlatingBrass
+  isSubfloor: false
+  deconstructTools: [ Prying ]
+  footstepSounds:
+    collection: FootstepHull
+  itemDrop: FloorTileItemBrassFilled
+  heatCapacity: 10000
+  
+- type: tile
+  id: FloorBrassFilledOnPlating
+  name: tiles-brass-floor-filled-onplate
+  parent: FloorBrassFilled
+  baseTurf: Plating
+  
+- type: tile
+  id: FloorBrassReebe
+  name: tiles-brass-floor-reebe
+  sprite: /Textures/Tiles/Misc/clockwork/reebe.png
+  baseTurf: PlatingBrass
+  isSubfloor: false
+  deconstructTools: [ Prying ]
+  footstepSounds:
+    collection: FootstepHull
+  itemDrop: FloorTileItemBrassReebe
+  heatCapacity: 10000
+  
+- type: tile
```

Comment:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/23f20b74-76b6-4b39-94b5-2b12c77a3377)

it's a very bad decision. I realize it's an easy way out, but you need a complete system that allows you to put any kind of tile on any kind of flooring. Making duplicate tiles on different surfaces is a conceptually terrible solution!

I'd like to hear what other maintainers have to say on this topic.

For now, I suggest removing the plating, and leaving only 2 kinds of new tiles.

Reply:
removed 👍